lib/src/main/java/com/auth0/jwt/interfaces/Verification.java

```diff
@@ -160,15 +160,18 @@
      */
     Verification withArrayClaim(String name, Integer... items) throws IllegalArgumentException;
 
-    /**
-     * Skip the Issued At ("iat") date verification. By default, the verification is performed.
-     */
-    Verification ignoreIssuedAt();
-
     /**
      * Creates a new and reusable instance of the JWTVerifier with the configuration already provided.
      *
      * @return a new JWTVerifier instance.
      */
+    Verification withArrayClaim(String name, Long ... items) throws IllegalArgumentException;
```

Comment:
This change swapped the javadoc lines. 

- Make sure the new `withArrayClaim()` method has something like this:

```
/**
     * Require a specific Array Claim to contain at least the given items.
     *
     * @param name  the Claim's name.
     * @param items the items the Claim must contain.
     * @return this same Verification instance.
     * @throws IllegalArgumentException if the name is null.
     */
```

- Make sure the `build()` method keeps this:
```
/**
     * Creates a new and reusable instance of the JWTVerifier with the configuration already provided.
     *
     * @return a new JWTVerifier instance.
     */
```

Reply:
Fixed